### PR TITLE
Standardize capitalization

### PR DESCRIPTION
**Pull request by vihdzp**
I've attempted to standardize capitalization and punctuation, both so the site looks better, and so that our translations are more uniform.

Basically, any small prompts or short text that would be found on a button does not end on a period. Any longer text, particularly information that would appear on a textbox, had punctuation added. Capital letters are only used for the first word in a sentence or prompt.

Further discussion on what things to capitalize/punctuate or not is welcome.

### CODE REVIEW

File: client/src/translation/en.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
...
     "ourMission": "Our mission",
-    "missionStatement": "<1>The music industry does not work for musicians or listeners and needs a radical re-imagination. </1><1>Mirlo is a community of musicians, listeners, and coders who are daring to do just that: taking lessons learned in working in the solidarity economy and applying them to our process and product.</1><1>We are building an online audio distribution (think Bandcamp) and patronage (think Patreon) platform that aims to be radical, accessible, open source (free & libre), modular, and standards based.</1>",
+    "missionStatement": "<1>The music industry does not work for musicians or listeners and needs a radical re-imagination. </1><1>Mirlo is a community of musicians, listeners, and coders who are daring to do just that, taking lessons learned in working in the solidarity economy and applying them to our process and product.</1><1>We are building an online audio distribution (think Bandcamp) and patronage (think Patreon) platform that aims to be radical, accessible, open source (free & libre), modular, and standards based.</1>",
```
Comment (simonv3) [inline client/src/translation/en.json:33]: colon is correct here I think?

`We're doing just that: things we're doing`

**Review by simonv3 — APPROVED** [review]
Looks good! Just the colon thing but that's minor

File: client/src/translation/en.json (hunk 1 above)
Comment (simonv3) [inline client/src/translation/en.json:33]:
```suggestion
    "missionStatement": "<1>The music industry does not work for musicians or listeners and needs a radical re-imagination. </1><1>Mirlo is a community of musicians, listeners, and coders who are daring to do just that: taking lessons learned in working in the solidarity economy and applying them to our process and product.</1><1>We are building an online audio distribution (think Bandcamp) and patronage (think Patreon) platform that aims to be radical, accessible, open source (free & libre), modular, and standards based.</1>",
```

**Review by simonv3 — APPROVED** [review]

File: client/src/translation/en.json (hunk 1 above)
Comment (vihdzp) [reply client/src/translation/en.json]: I think the sentence is instead trying to imply:
`We're doing just that, how we're doing it`
It's the next sentence that explains what we're doing.

**Pull request merged**